google/ggrc-core
# GGRC-6673 Disable possibility to add/edit custom attributes in administration

# Issue description

Disable adding/removing/updating global custom attributes for controls on administration dashboard.

# Steps to test the changes

1. Open 'Custom Attributes' tab in 'Admin Dashboard'
2. Expand 'Contols'
3. Make sure icon to add custom attributes is removed
4. Make sure 'Edit' column is removed

# Solution description

Control's model flag was introduced previously. Based on this flag functionality to add/update custom attributes is disabled.

# Sanity checklist




# PR Review checklist
